src/app/shared/navbar/_navbar-theme.scss

```diff
@@ -13,6 +13,15 @@
 
     .docs-navbar, .docs-navbar-header {
       background: mat.get-color-from-palette($primary);
+
+      & a:hover, & a:focus {
+        background: rgba(0, 0, 0, 0.4);
+        color: mat.get-color-from-palette($accent, 200);
```

Comment:
I just want to make sure that `200` here isn't too dark to meet the contrast guidelines.

Reply:
What was the purpose of using the accent color here instead of just inheriting the color? Was the shade of the background not enough to differentiate selected versus hovered/focus states? or was there a contrast issue?

Reply:
> What was the purpose of using the accent color here instead of just inheriting the color? Was the shade of the background not enough to differentiate selected versus hovered/focus states? or was there a contrast issue?

Used only to differentiate, similar to the sidebar. but by removing it, there is no problem of contrast.